.github/workflows/iso.yml

```diff
@@ -3,6 +3,7 @@ on:
   pull_request:
     paths:
       - "deploy/iso/**"
+      - ".github/workflows/**"
```

Comment:
how about changing this to only the  .github/workflows/iso.yml 

so it doesn't get trigered when we edit the other workflows ?

Reply:
good idea